compiler/src/iree/compiler/Codegen/Common/IREEComprehensiveBufferizePass.cpp

```diff
@@ -241,6 +260,19 @@ void IREEComprehensiveBufferizePass::runOnOperation() {
   }
 }
 
+/// Run constant bufferize.
+void IREEBufferizeConstantsPass::runOnOperation() {
+  mlir::bufferization::OneShotBufferizationOptions opt;
+  opt.copyBeforeWrite = true;
+  opt.enforceAliasingInvariants = false;
+  opt.opFilter.allowOperation(arith::ConstantOp::getOperationName());
+  if (failed(mlir::bufferization::runOneShotBufferize(getOperation(), opt,
+                                                      nullptr))) {
```

Comment:
nit: add function argument comment for magic values.

```suggestion
                                                      /*statistics=*/nullptr))) {
```